fusedrug/data/tokenizer/modulartokenizer/test_multi_tokenizer_creation.py

```diff
@@ -258,9 +258,7 @@ def get_training_corpus(dataset: List) -> Generator:
     print("Fin")
 
 
-@hydra.main(
-    config_path="./configs", config_name="tokenizer_config_personal", version_base=None
-)
+@hydra.main(config_path="./configs", config_name="tokenizer_config", version_base=None)
```

Comment:
@floccinauc 
Just a thought
Maybe we should rename this file to bmfm_bio_modular_tokenizer and move the content to bmfm-bio?